app/renderer/components/preferences/syncTab.js

```diff
@@ -155,23 +219,49 @@ class SyncTab extends ImmutableComponent {
     ]
     return this.props.syncPassphraseVisible
       ? <div>
-        <Button l10nId='syncHidePassphrase' className='whiteButton wideButton syncToggleButton' onClick={this.props.hidePassphrase} />
-        <pre id='syncPassphrase'>{words.join('\n')}</pre>
+        <ul className={css(styles.syncOverlayBody__listWrapper)}>
+          <li className={css(styles.syncOverlayBody__listItem, commonStyles.noMarginBottom)}>
+            <Button className='whiteButton'
+              l10nId='syncHidePassphrase'
+              testId='syncHidePassphraseButton'
+              onClick={this.props.hidePassphrase} />
+            <pre data-test-id='syncPassphrase'
+              className={css(
+                styles.passphrase,
+                styles.listItem__passphrase,
+                commonStyles.noMarginBottom
+              )}>{words.join('\n')}</pre>
+          </li>
+        </ul>
       </div>
-      : <Button l10nId='syncShowPassphrase' className='whiteButton syncToggleButton' onClick={this.props.showPassphrase} />
+      : <ul className={css(styles.syncOverlayBody__listWrapper)}>
+        <li className={css(styles.syncOverlayBody__listItem, commonStyles.noMarginBottom)}>
+          <Button className='whiteButton'
+            l10nId='syncShowPassphrase'
+            testId='syncShowPassphraseButton'
+            onClick={this.props.showPassphrase} />
+        </li>
+      </ul>
   }
 
   get overlayContent () {
     return <div className='syncOverlay'>
-      <div>
+      <div className={css(styles.syncOverlayBody__wrapper)}>
         <ol>
-          <li data-l10n-id='syncNewDevice1' />
-          <li data-l10n-id='syncNewDevice2' />
+          <li className={css(styles.syncOverlayBody__listItem, commonStyles.noMarginTop)} data-l10n-id='syncNewDevice1' />
+          <li className={css(styles.syncOverlayBody__listItem)} data-l10n-id='syncNewDevice2' />
           {this.qrcodeContent}
```

Comment:
This function returns `ul`, so you need to add `li` wrap before calling this function